doc-content/enterprise-only/admin-and-config/custom-query-proc.adoc

```diff
@@ -0,0 +1,37 @@
+[id='custom-query-proc']
+= Running a custom query on the {PRODUCT} database
+
+You can use the `ExecuteSQLQueryCommand` executor command to run a custom query on the {PRODUCT} database, including the audit log data tables. You can set up a job that runs this command in {CENTRAL}.
+
+.Procedure
+. In {CENTRAL}, go to *Manage* > *Jobs*.
+. Click *New Job*.
+. Enter values for *Business Key*, *Due On*, and *Retries* fields.
+. Enter the following command into the *Type* field.
++
+[source]
+----
+org.jbpm.executor.commands.ExecuteSQLQueryCommand
+----
++
+. To configure parameters, complete the following steps:
++
+--
+.. Open the *Advanced* tab.
+.. Click *Add Parameter*.
+.. Enter a parameter in the *Key* column and enter a parameter value in the *Value* column.
+--
++
+For the list of parameters for the command, see xref:custom-query-params-ref[].
++
+. Click *Create*. {CENTRAL} creates the custom query job.
+. Optional: If you want to retrieve the results of the query, complete the following steps:
+.. In the list of jobs that {CENTRAL} displays, find the job that you started. If the job is not present in the list, remove any filters in the *Active filters* list.
+.. Record the *id* value for the job.
+.. Using a web browser, access the Swagger documentation on your {KIE_SERVER} at `\<kie_server_address>/docs`, for example, `http://localhost:8080/kie-server/docs/`.
+.. Click the `GET /server/jobs/{jobId}` request.
+.. In the *jobId* parameter field, enter the *id* value that you recorded.
+.. Set the *withErrors* parameter to `true`.
```

Comment:
What type of Ui element is the parameter. If it is a radio button or list, follow the [radio button style](https://www.ibm.com/docs/en/ibm-style?topic=elements-ui#radio_button) or [list style](https://www.ibm.com/docs/en/ibm-style?topic=elements-ui#list).